examples/fluids/problems/blasius.c

```diff
@@ -162,31 +162,40 @@ PetscErrorCode NS_BLASIUS(ProblemData *problem, DM dm, void *ctx) {
   // ------------------------------------------------------
   //               SET UP Blasius
   // ------------------------------------------------------
-  problem->ics.qfunction                       = ICsBlasius;
-  problem->ics.qfunction_loc                   = ICsBlasius_loc;
-
-  CeedScalar Uinf   = 40;          // m/s
-  CeedScalar delta0 = 4.2e-4;      // m
-  CeedScalar theta0 = 288.;        // K
-  CeedScalar P0     = 1.01e5;      // Pa
-  PetscBool  weakT  = PETSC_FALSE; // weak density or temperature
-  PetscReal  mesh_refine_height = 5.9e-4; // m
-  PetscReal  mesh_growth        = 1.08;   // [-]
-  PetscInt   mesh_Ndelta        = 45;     // [-]
-  PetscReal  mesh_top_angle     = 5;      // degrees
+  problem->ics.qfunction        = ICsBlasius;
+  problem->ics.qfunction_loc    = ICsBlasius_loc;
+
+  CeedScalar Uinf               = 40;          // m/s
+  CeedScalar Tinf               = 288.;        // K
+  CeedScalar T_wall             = 400.;        // K
+  CeedScalar delta0             = 4.2e-3;      // m
+  CeedScalar theta0             = 288.;        // K
+  CeedScalar P0                 = 1.01e5;      // Pa
+  CeedInt    N                  = 10;          // Number of Chebyshev terms
```

Comment:
I think this may not be enough, especially at higher Mach or if $\eta_\max$ is larger. I think we were pretty happy with 20 through a reasonable range. Solution evaluation is cheap in any case, but I'd be inclined to bump this up to avoid users being surprised.
```suggestion
  CeedInt    N                  = 20;          // Number of Chebyshev terms
```